src/actions/authActions.ts

```diff
@@ -1,6 +1,9 @@
 import {ThunkAction} from 'redux-thunk';
 import fireAuth from '../config/firebaseConfig';
 import {RootState} from '../store';
+// @ts-ignore
```

Comment:
What was typescript doing here?

Reply:
So @type imports are needed, but an @type module doesn't exist for ReactRouterGlobalHistory. I had no choice but to add that.